src/form-styles/form-styles-multi-step.ts

```diff
@@ -0,0 +1,157 @@
+export class formStylesMultiStep  {
+    constructor() {
+        this.addStyles();
+    }
+
+    private addStyles(): void {
```

Comment:
raczej bym zrobil metode generateStyles ktora zwraca stringa ze stylami. 
I druga metoda addStyles by brala wartosc z tamtej metody i robila to co robi. 
Bo tej metody praktycznie nie da sie przetestowac jednostkowo